multi-agent/adapters/outbound/mongo/blueprint_repository.py

```
@@ -1,8 +1,10 @@
 import pymongo
+from pymongo.errors import DuplicateKeyError
 from uuid import uuid4
 from datetime import datetime, timezone
 from typing import List, Dict, Any, Optional
```

Ruff (0.15.21)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)